src/ffi/tests.rs

```diff
@@ -79,7 +79,7 @@ fn test_timezone_from_offset() {
     use crate::types::PyDelta;
 
     Python::with_gil(|py| {
-        let delta = PyDelta::new(py, 0, 100, 0, false).unwrap();
+        let delta = PyDelta::new_bound(py, 0, 100, 0, false).unwrap();
         let tz: &PyAny = unsafe { py.from_borrowed_ptr(PyTimeZone_FromOffset(delta.as_ptr())) };
```

Comment:
Maybe we can remove this gil-ref (and the one below) as well, while we're here.

Reply:
Good idea. Reading this code I see they're also bugged. [`PyTimeZone_FromOffset`](https://docs.python.org/3/c-api/datetime.html#c.PyTimeZone_FromOffset) is documented to return a new reference, so I'll change this to `.assume_owned(py)`.

Reply:
Perfekt 👍